.buildkite/bk.integration.pipeline.yml

```diff
@@ -21,6 +21,74 @@ steps:
       image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-beats-ci-with-hooks:0.5"
       useCustomGlobalHooks: true
 
+  - label: Start Serverless project
+    key: integration-serverless
+    env:
+      ASDF_TERRAFORM_VERSION: 1.9.2
+    command: |
+      #!/usr/bin/env bash
+      set -euo pipefail
```

Comment:
Nit/question: do we need these here?  I see that this is the same pattern used in the `integration-ess` step but I don't see it used in other BK pipelines that source a script.

Reply:
I definitely got "inspired" (😄) by the existing `integration-ess` but AFAICT both `ess_start.sh` and `serverless_start.sh` do start with the same lines, so these are not required. @pazone I vote to remove these wdyt? 🙂

Reply:
It's deleted here https://github.com/elastic/elastic-agent/pull/7441